app/views/layouts/mailer.text.erb

```diff
@@ -1,5 +1,5 @@
 <%= yield %>
 ---
 
-<%= t('application_mailer.signature', instance: site_hostname) %>
+<%= t 'about.hosted_on', domain: site_hostname %>
```

Comment:
No parentheses?

Reply:
I'm not sure what you're asking... That's valid ruby syntax. Code style wise we don't really have a rule either way, but I figured in templates I wanna keep stuff as short as possible.

Reply:
Yeah, it seems mixed in the rest of the codebase. I just called it out here because you removed them. :smile:

Reply:
That was more about me copypasting the line from the HTML template since I want the plaintext and HTML versions to use the same strings (I will have a separate PR cleaning up the remaining user_mailer text templates)